src/calculation/HTMLBuilder.ts

```diff
@@ -243,19 +243,11 @@ export function generateClauseCoverageHTML(
     const flattenedStatementResults = detailedResults.flatMap(s => s.statementResults);
```

Comment:
```suggestion
    const flattenedStatementResults = detailedResults[0].statementResults;
```

We could work from the statementResults from just the first patient. Since we just need the names of the statements, and wether or not relevance is `NA`. If things are NA then the statement is not use at all by the measure. This information is on all statementResults for all patients in all groups (unique results to the group of course).